### ISSUE TITLE
Missing level 1 headers accessibility issue [Not required for WCAG conformance]

### ISSUE THREAD

**Issue by josenino95** [carpentries/varnish#197]
Related to my recent issues opened, SiteImprove flagged that some of the auto-generated pages don't have a level 1 header `h1`. In particular, these are the six pages flagged:

- aio.html
- instructor/aio.html
- images.html
- instructor/images.html
- key-points.html
- instructor/key-points.html

**However, I should highlight that SiteImprove says "This is a best practice only and not currently required for WCAG conformance."** But in case it's an easy fix, it would be nice to have level one headers for this pages, as it also makes navigation of the page easier for all users.

In [this link you'll find the SiteImprove report](https://my2.siteimprove.com/Inspector/1237935/A11Y/Page?PageId=343679223941&impmd=A74FC28FD5FAD95A0E08DD9E49865810&issue=cantTell+failed&conf=a+aa+aria+si#/sia-r61/failed/eEVTNU4yQVg=;NjgxNGViNjc=) of the issue for [our fork of the Git lesson](https://carpentry.library.ucsb.edu/2025-04-14-git-novice/key-points.html), specifically the key-points. If that link doesn't work for you because it requests you to login, here is what it shows me.

<details>

> Page does not start with a level 1 heading
> Issue
> Accessibility best practices
> A level 1 heading h1 tells the user what the page is about before they decide to navigate through the content.
> 
> Headings are used to structure content into a hierarchy of importance. If a page starts with a different heading level, it can cause confusion for the user.
> 
> Why is this an issue?
> Heading tags help assistive technology users to understand how content is structured. They're also used for in-page screen reader navigation.
> 
> When determining heading levels, you should ensure that they follow a nested structure. h1 is used for the primary heading, followed by h2 for subheadings, and so on to h6.
> 
> Correcting your heading structure may also help to improve your site's SEO.
> 
> This is a best practice only and not currently required for WCAG conformance.
> Help Center: [Page does not start with a level 1 heading, explained](https://help.siteimprove.com/support/solutions/articles/80001058800-accessibility-rule-page-does-not-start-with-a-level-1-heading-explained)
> Siteimprove Alfa: [Technical documentation for rule SIA-R61](https://alfa.siteimprove.com/rules/sia-r61)

> 
> Occurrences
> 1
> 
> Occurrence details:Entire document
> Use h1 for the main heading on the page.
> First heading: h2
> Heading level: 2

</details>
